src/components/Icons/Svg.tsx

```
@@ -0,0 +1,67 @@
+  };
+
+  return (
+    <div css={svgWrapperStyle(width, height, size)}>
+      <svg
+        css={svgStyle}
+        fill={fill}
+        color={color ? theme.colors[color] : 'currentColor'}
+        viewBox={viewBox}
+        xmlns="http://www.w3.org/2000/svg"
+        style={{transform: `rotate(${getRotation()})`}}
+        dangerouslySetInnerHTML={{__html: childrenString}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)